src/sbt-test/sbt-rat/apache-headers/build.sbt

```diff
@@ -1,2 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I mean technically we're developing this on my GitHub instead of at Apache. Though I believe you'll need to update the NOTICE/COPYRIGHT files if you're bringing in outside code (like the .java file earlier).

Reply:
Yep, I originally intended this to be an RFC and figured this this file would be moved to rat-core and wouldn't be included anymore, but that might take some effort/time. I'll update the licenses appropriately.

Reply:
I'm hoping to work on integrating this upstream to Creadur at some point, so we'll be granting it back to them anyways.

Reply:
Actually, the NOTICE of this plugin currently has the standard ASF notice:
```
This product includes software developed at
The Apache Software Foundation (https://www.apache.org/)
```
My understanding is that since the both pieces of software are under the Apache license and since both have the standard "developed at ASF" notice without anything extra, then it isn't necessary to list all the the notice again. So maybe we don't need to modify anything for this file?

Reply:
Right, the notice is fine since it has the notice already.